lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
```

Comment:
Right now I was thinking if we could drop the whole `share_group`. You could but validating if the `share_group` adds up to a 100% should be part of the ETSource validation. And even than I'm not sure if that should be the case. Somebody - in my opinion must have the freedom to - set 3 out of 10 shares; for example; and leave the rest blank.